apps/callscreen/test/unit/handled_call_test.js

```diff
@@ -172,7 +174,8 @@ suite('dialer/handled_call', function() {
       });
 
       test('should have a numberNode in a numberWrapper', function() {
```

Comment:
This test seems fairly arbitrary, but we should add another one for ```outerNode``` if we want to be consistent.